_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+  [1.10.1/1.11.0](https://github.com/kubeflow/manifests/issues/3038) milestones
+
+| Notebooks | Dashboard | Pipelines | Katib | Trainer | KServe | Model Registry | Spark |
+|:---------:|:---------:|:---------:|:-----:|:-------:|:------:|:--------------:|:-----:|
+| [1.10](https://github.com/kubeflow/kubeflow/issues/7459)      | [1.10](https://github.com/kubeflow/kubeflow/tags)      | [2.4.1](https://github.com/kubeflow/pipelines/releases)     | [0.18](https://github.com/kubeflow/katib/releases)  | [1.9](https://github.com/kubeflow/trainer/releases)     | [0.14](https://github.com/kserve/kserve/releases)   | [0.2.15](https://github.com/kubeflow/model-registry/releases)         | [2.1.0](https://github.com/kubeflow/spark-operator/releases/tag/v2.1.0) |
+
+| Kubernetes | Kind | Kustomize | Cert Manager | Knative | Istio | Dex  | Oauth2-proxy |
+|:----------:|:----:|:---------:|:------------:|:-------:|:-----:|:----:|:------------:|
+| 1.31-1.33  | 0.26 | 5.4.3     | 1.16.1       | 1.16    | 1.24  | 2.41 | 7.7          |
+
+### Security:
+
+* CVE reductions - regular scanning with trivy
+* Kubernetes and container security best practices:
+    * [Rootless containers](https://github.com/kubeflow/manifests/issues/2528) / PodSecurityStandards restricted for:
+      Istio-CNI, Knative, Dex, Oauth2-proxy, Spark
+    * [50 % done](https://github.com/kubeflow/manifests/pull/3050): KFP, Notebooks / Workspaces, Katib, Trainer, Kserve, …
+    * Istio-CNI as default for rootless Kubeflow postponed to [1.10.1](https://github.com/kubeflow/manifests/milestone/2)
+* OIDC-authservice has been replaced by oauth2-proxy
+* [Oauth2-proxy](https://github.com/kubeflow/manifests#oauth2-proxy) and [Dex](https://github.com/kubeflow/manifests#dex)
+  documentation for external OIDC authentication (Keycloak, and OIDC providers such as Azure, Google etc.)
+
+Trivy CVE scans March 25 2025:
+
+|     Working Group      | Images | Critical CVE | High CVE | Medium CVE | Low CVE |
+|:----------------------:|:------:|:------------:|:--------:|:----------:|:-------:|
+|         Katib          |   17   |      11      |   101    |    417     |   734   |
+|       Pipelines        |   15   |      57      |   490    |    4030    |   1922  |
+| Workbenches(Notebooks) |   12   |      12      |    59    |    179     |   224   |
+|         Kserve         |   16   |      21      |   305    |    6803    |   1588  |
+|       Manifests        |   14   |      8       |    4     |     94     |    52   |
+|        Trainer         |   1    |      0       |    0     |     1      |    0    |
+|     Model Registry     |   6    |      1       |    13    |    153     |   188   |
+|         Spark          |   1    |      5       |    37    |    1640    |   141   |
+|       All Images       |   81   |     115      |   1009   |   13275    |   4804  |
+
+## Pipelines
+
+Kubeflow Pipelines 2.4.1 introduces several enhancements to improve the flexibility and control of your machine learning
+workflows.
+
```

Comment:
I know we are trying to keep this short... but pipelines is the most popular app.    How much time will this potentially save a MLOps engineer ?   Minutes or Hours ?     It would be nice to quantify the benefit..     Kubeflow Pipelines 2.4.1 introduces several enhancements to improve the flexibility and control of your machine learning
workflows.   This can save hours of work when defining new pipeline definitions.

Reply:
yes, I'll remove this paragraph. + we already provided some high level benefits above